hack/gen-swagger-doc/deploy.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/bash
+
+set -e
+
+GITHUB_FQDN=github.com
+API_REF_REPO=kubevirt-incubator/api-reference
+API_REF_DIR=/tmp/api-reference
+
+git clone \
+    "https://${API_REFERENCE_PUSH_TOKEN}@${GITHUB_FQDN}/${API_REF_REPO}.git" \
+    "${API_REF_DIR}"
+cp -f hack/gen-swagger-doc/html5/content/*.html "${API_REF_DIR}/content/"
+
+cd "${API_REF_DIR}"
+
+git config --global user.email "travis@travis-ci.org"
+git config --global user.name "Travis CI"
+
+if git status --porcelain | grep --quiet "^ M" ;
+then
+  git add content/*.html
```

Comment:
I think that you will need a `git add -A content/*.html` to completely sunchronize the directory.

Reply:
Done